docs/models/files.md

```diff
@@ -0,0 +1,40 @@
+Model Files
+===========
+
+A "model" is simply a shape. It can be a simple cube, it can be several cubes, it can be a truncated icosidodecahedron, or anything in between. Most models you'll see will be in the vanilla JSON format. Models in other formats are loaded into `IModel`s by an `ICustomModelLoader` at runtime. Forge provides default implementations for WaveFront OBJ files and Blitz3D files. Most things do not care about what loaded the model or what format it's in as they all implement a common interface `IModel` in code.
+
+Whenever you refer to a model using a `ResourceLocation`, the path is normally relative to `models` (e.g. `examplemod:block/block` → `assets/examplemod/models/block/block`). A notable exception is within a [blockstate JSON][], where model paths are relative to `models/block` (e.g. `examplemod:block` → `assets/examplemod/models/block/block`).
+
+Block and item models differ in a few ways, the major one being [item property overrides][overrides].
+
+When referring to textures in models, keep a few things in mind. First, you reference textures as RLs relative to `textures/`. (E.g. `examplemod:blocks/test` → `assets/examplemod/textures/blocks/test.png`.) Second, when specifying [UV coordinates][UV], (0,0) is taken to mean the **top-left** corner. UVs are *always* from 0 to 16. If the texture is larger or smaller, it is scaled to fit. A texture must also be square, and its side length must be a power of two (e.g. 1x1, 2x2, 4x4, 8x8, but not 4x8). If an mcmeta file is present, the image can be rectangular in order to fit the extra image data, but the final texture as seen externally is still a square (e.g. the command block texture is animated and the on-disk file is rectangular, but the model uses UVs (0,0) to (16,16) because the texture is still seen as square.)
```

Comment:
a bit unclear at first read 
Maybe replace the last super long sentence with something like

"If the texture is animated, the image can be rectangular, and is interpreted as sequential regions of UV(0,0) to UV(16,16) from top to bottom"